### ISSUE TITLE
importing video 720p240 (worked in debian stable / 14.3 stopped working in 2.4.1

### ISSUE THREAD

**Issue by gbu-censhare** [OpenShot/openshot-qt#1585]
Hi,

when importing a video from my Yi 4K+ recorded in 720p240 2.4.1 only shows a still image, videos in 720p30 from this camera work fine. 

i tried from the PPA (with Debian Stretch) & the version in Debian Buster (amd64). 

**System Details (please complete the following information):**
 - Operating System / Distro: Debian Buster (amd64)
 - OpenShot Version 2.4.1

**To Reproduce**
1. Import Video file (example attached)
2. Add it to the timlime
3. Move position on timeline
4. Watch Preview Window 

**Expected behavior**
Video Content should change while moving playing/ moving the marker on the timeline

**Screenshots**

**Logs**
If you are experiencing a crash, please collect and attach logs of the problem.

**Additional context**

**Comment by gbu-censhare** [comment]
[720p30.MP4.zip](https://github.com/OpenShot/openshot-qt/files/1998308/720p30.MP4.zip)

**Comment by gbu-censhare** [comment]
![screen2](https://user-images.githubusercontent.com/12797061/39965972-fe87a9a0-56a3-11e8-8818-a81333a2f500.png)
![screen1](https://user-images.githubusercontent.com/12797061/39965973-fea66d04-56a3-11e8-8eb0-03b3bf16475b.png)
![screen3](https://user-images.githubusercontent.com/12797061/39965974-fec0eab2-56a3-11e8-95da-43afb0a7aa71.png)
![screen4](https://user-images.githubusercontent.com/12797061/39965975-fedccfca-56a3-11e8-8c17-144d83cacc2c.png)

**Comment by gbu-censhare** [comment]
[720p240.MP4.zip](https://github.com/OpenShot/openshot-qt/files/1998312/720p240.MP4.zip)

Sorry conntet is a little different, due to size limitations but the issue is the same

**Comment by peanutbutterandcrackers** [comment]
Hey there,

Thank you for the really great bug report. You have included the video too; this will help others help you.

In the meantime, I would like to suggest that you [switch to the daily builds](https://gist.github.com/peanutbutterandcrackers/3618493b99ba08003e4e4b1602459914). Perhaps it'll work in the latest daily version.

**Comment by N3WWN** [comment]
I found the issue with 240 fps video:

As a sanity check, libopenshot checks for an unreasonably high frame rate.  When this check was written, it appears that 120 fps frame rates were the highest frame rates available in standard equipment, so any frame rate high than that was flagged as invalid.  

Since we now have 240 fps video from many cameras, I've submitted a PR which increases the valid frame rates to 240 fps.
